aten/src/ATen/autocast_mode.cpp

```diff
@@ -330,6 +330,8 @@ TORCH_LIBRARY_IMPL(aten, Autocast, m) {
   KERNEL_CUDA(cdist, fp32)
   KERNEL_CUDA(renorm, fp32)
   KERNEL_CUDA(logsumexp, fp32)
+  KERNEL_CUDA(upsample, fp32)
+  KERNEL_CUDA2(upsample, fp32)
```

Comment:
Id on't think you need this one. This is for when you want to specify a specific overload.

Reply:
This one? Do you mean CUDA?

Reply:
The CUDA2 yes. I expect this is not going to compile

Reply:
Done